src/node_crypto.h

```diff
@@ -478,19 +479,16 @@ class SignBase : public BaseObject {
   } Error;
 
   SignBase(Environment* env, v8::Local<v8::Object> wrap)
-      : BaseObject(env, wrap),
-        mdctx_(nullptr) {
+      : BaseObject(env, wrap) {
   }
 
-  ~SignBase() override;
-
   Error Init(const char* sign_type);
   Error Update(const char* data, int len);
 
  protected:
   void CheckThrow(Error error);
 
-  EVP_MD_CTX* mdctx_;
+  DeleteFnPtr<EVP_MD_CTX, EVP_MD_CTX_free> mdctx_;
```

Comment:
Is there a reason not to reuse `EVPMDPointer`?

Reply:
@tniessen thanks for catching, updated!